test/check-machines-create

```diff
@@ -1633,8 +1641,10 @@ vnc_password= "{vnc_passwd}"
             root.find('os').find('resources').find('minimum').find('storage').text = '134217728'
             root.find('os').find('eol-date').text = '9999-12-31'
             new_fedora_28_xml = ET.tostring(root)
-            self.machine.execute(f"echo '{str(new_fedora_28_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/fedora-28.xml")
-            self.machine.execute(f"mount -o bind  {self.test_obj.vm_tmpdir}/fedora-28.xml /usr/share/osinfo/os/fedoraproject.org/fedora-28.xml")
+            self.machine.execute(f"""
+            echo '{str(new_fedora_28_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/fedora-28.xml
```

Comment:
In your next step: Split this up and use self.machine.write()

Reply:
```
test/check-machines-consoles:        self.machine.execute(f"echo '' > {args['logfile']}")
test/check-machines-create:            self.machine.execute("echo >>/etc/hosts 127.0.0.42 mirror.i3d.net")
test/check-machines-create:            self.machine.execute("echo >>/etc/hosts ::1 mirror.i3d.net")
test/check-machines-create:            self.machine.execute(f"echo '{str(new_rhel_8_1_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/rhel-8.1.xml")
test/check-machines-create:            self.machine.execute(f"echo '{str(new_rhel_8_2_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/rhel-8.2.xml")
test/check-machines-create:            self.machine.execute(f"echo '{str(new_rhel_7_1_xml, 'utf-8')}' > {self.test_obj.vm_tmpdir}/rhel-7.1.xml")
test/check-machines-lifecycle:        self.machine.execute(f"echo '{user_name}:foobar' | chpasswd")
test/check-machines-lifecycle:        self.machine.execute(f"echo '' > {args['logfile']}")
test/check-machines-lifecycle:        self.machine.execute(f"echo '' > {args2['logfile']}")
```

Note, we have more places we can use `self.machine.write`

Reply:
I assume you mean `self.write_file`.

Reply:
No, that will try to restore the file, but it's already in the vm_tmpdir.

Reply:
Didn't read into the context, but it seems that we can generally cleanup the `execute('echo ...')` pattern.